src/output.jl

```diff
@@ -106,10 +106,17 @@ end
 
 const _extension_map = Dict("tikz" => "tex")
 
-function addExtension(fn::AbstractString, ext::AbstractString)
+"""
+    addExtension(filepath, extension)
+
+Change filepath extension according to the extension map
+"""
+function addExtension(fp, ext::AbstractString)
+    parent = dirname(fp)
+    fn = basename(fp)
     oldfn, oldext = splitext(fn)
```

Comment:
```suggestion
    _, oldext = splitext(fn)
```

See https://docs.julialang.org/en/v1/manual/variables/#man-allowed-variable-names (underscore convention when [`l-value`](https://en.wikipedia.org/wiki/Value_(computer_science)#lrvalue))